dts/bindings/gpio/seeed-xiao-header.yaml

```
@@ -0,0 +1,33 @@
+# Copyright (c) 2019 Foundries.io
+# Copyright (C) 2019 Peter Bigot Consulting, LLC
+# Copyright (C) 2021 Peter Johanson
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    GPIO pins exposed on Seeeduino Xiao (and compatible devices) headers.
+
+    The Seeeeduino Xiao layout provides two headers, along opposite
+    edges of the board.
+
+    Proceeding counter-clockwise:
+    * A 7-pin Digital/Analog Input header.  This has input signals
+      labeled from 0 at the top through 6 at the bottom.
+    * An 7-pin header Power and Digital/Analog Input header.  This
```

```suggestion
    * A 7-pin header Power and Digital/Analog Input header.  This
```

feel free to fix if you have time, not blocking